roboflow/models/instance_segmentation.py

```diff
@@ -0,0 +1,72 @@
+import io
+import urllib
+
+import requests
+from PIL import Image
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+
+from roboflow.config import INSTANCE_SEGMENTATION_MODEL, INSTANCE_SEGMENTATION_URL
+from roboflow.util.image_utils import validate_image_path
+from roboflow.util.prediction import PredictionGroup
+
+
+class InstanceSegmentationModel:
+    def __init__(
+        self,
+        api_key,
+        version_id,
+    ):
+        """
+        :param api_key: Your API key (obtained via your workspace API settings page)
+        :param version_id: The ID of the dataset version to use for predicting
+        """
+        self.__api_key = api_key
+        self.id = version_id
+
+        version_info = self.id.rsplit("/")
+        dataset_id = version_info[1]
+        version = version_info[2]
+
+        self.api_url = f"{INSTANCE_SEGMENTATION_URL}/{dataset_id}/{version}"
+
+    def predict(self, image_path):
+        """
+        Infers detections based on image from specified model and image path
+
+        :param image_path: Path to image (can be local or hosted)
+        :return: PredictionGroup --> a group of predictions based on Roboflow JSON response
+        """
```

Comment:
The `predict()` method on the other models accepted a `hosted` flag that seemed a little redundant. We can determine if something is hosted or not by looking at the `image_path`, so I omitted it here. We can maybe deprecate this argument on the other models as well.